docs/source/dev-guide/deep-dive-install.md

```diff
@@ -0,0 +1,648 @@
+5. Pre-link scripts are run for Python 3.9.9 and numpy 1.19, if any
+6. Files in the Python 3.9.9 and numpy 1.19 packages are linked and/or copied to the prefix. This
+   can be parallelized.
+7. Entry points are created for the new packages, if any.
+8. Post-link scripts are run.
+9. `pyc` files are generated for the new packages.
+10. The new packages are registered under `conda-meta/`.
+11. The menu shortcuts are created for the new packages, if any.
+
+Any of these steps can fail with a given exception. If that's the case, the first of those
+exceptions is printed to STDOUT. Additionally, if `rollback_enabled` is properly configured in
+the `context`, the transaction will be rolled back by calling the `.reverse()` method in each
+action, from last to first.
+
+If no exceptions are reported, then the actions can run their cleanup routines.
+
+And that's it! If this command had resulted in a new environment being created, you would get a
+message telling you how to activate the newly created environment.
+
+## Conclusion
+
+This is what happens when you type `conda install`. It might be a bit more involved than you
+initially thought but it all boil downs to only some steps. TLDR:
+
+1. Parse arguments and initialize the context
+2. Download and build the index
+3. Tell the solver what we want
+4. Convert the solution into a transaction
+5. Verify and run each action contained in the transaction
+
+
+<!-- Links and references -->
+[conda_build_channels]: https://docs.conda.io/projects/conda-build/en/latest/concepts/generating-index.html
+[conda_build_package_names]: https://docs.conda.io/projects/conda-build/en/latest/concepts/package-naming-conv.html
+[conda_cli_install_presolve_logic]: https://github.com/conda/conda/blob/4.11.0/conda/cli/install.py#L107
+[conda_install_delegation]: https://github.com/conda/conda/blob/4.11.0/conda/cli/conda_argparse.py#L775
+[conda_main_install_delegation]: https://github.com/conda/conda/blob/4.11.0/conda/cli/main_install.py#L12
+[conda_shell_function]: https://github.com/conda/conda/blob/4.11.0/conda/shell/etc/profile.d/conda.sh#L62-L76
+[conda.activate]: https://github.com/conda/conda/blob/4.11.0/conda/activate.py
+[conda.cli.conda_argparse:do_call]: https://github.com/conda/conda/blob/4.11.0/conda/cli/conda_argparse.py#L77
+[conda.cli.conda_argparse:generate_parser]: https://github.com/conda/conda/blob/4.11.0/conda/cli/conda_argparse.py#L28
+[conda.cli.install:install]: https://github.com/conda/conda/blob/4.11.0/conda/cli/install.py#L107
+[conda.cli.main_install]: https://github.com/conda/conda/blob/4.11.0/conda/cli/main_install.py
+[conda.cli.main:main]: https://github.com/conda/conda/blob/4.11.0/conda/cli/main.py#L121
+[conda.cli]: https://github.com/conda/conda/tree/4.11.0/conda/cli
+[conda.core.index:get_reduced_index]: https://github.com/conda/conda/blob/4.11.0/conda/core/index.py#L246
+[conda.core.link:_prepare]: https://github.com/conda/conda/blob/4.11.0/conda/core/link.py#L266
```

Comment:
You've got two of these "conda.core.link:_prepare" links with the same name, but different links.